There is a problem with Mail::Audit versions up to and including 1.9, which
will cause mail to be rewritten as it passes through Mail::Audit. 

This patch fixes the issue.  Apply with:

	patch < MailAudit.patch



--- Audit.pm.bak    Wed Sep 12 16:25:51 2001
+++ Audit.pm        Wed Sep 12 16:31:47 2001
@@ -32,7 +32,8 @@
     my $self = bless({ 
         %opts, 
         obj => Mail::Internet->new(
-                    exists $opts{data}? $opts{data} : \*STDIN
+                    exists $opts{data}? $opts{data} : \*STDIN,
+                   'Modify' => 0
                 ) 
     }, $class) ;
     if (exists $self->{loglevel}) { 

